<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QDialog" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameDate">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string>CurrentDate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelYearsAgo">
        <property name="text">
         <string>Years ago:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSearchDate">
        <property name="text">
         <string>SearchDate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHas">
        <property name="text">
         <string>has</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMatchCount">
        <property name="text">
         <string>MatchCount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMatches">
        <property name="text">
         <string>matches</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonShowNext">
        <property name="text">
         <string>Show Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonShare">
        <property name="text">
         <string>Share</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameImage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelImagePlaceholder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelImageFolder">
        <property name="text">
         <string>Image Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditImageFolder">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
